truffle.js

```diff
@@ -104,7 +108,7 @@ module.exports = {
     },
     forked_live: {
       network_id: 72,
-      gas: 6800000, // close to current mainnet limit
+      gas: gasLimit,
```

Comment:
please keep old value on `forked` networks - they use different instance where gas floor is `6800000`, alternatively you can define another variable

you can also use same value for mainnet. I know that block limit is not much higher but we should not have any contract or transaction that goes above `forked` (staging) limit and we can safely enforce it.